app/models/authenticator/amazon.rb

```diff
@@ -115,15 +116,11 @@ def verify_credentials(access_key_id, secret_access_key)
     end
 
     def aws_connect(access_key_id, secret_access_key, service = :IAM)
-      require 'aws-sdk-v1'
-
-      AWS.const_get(service).new(
-        :access_key_id     => access_key_id,
-        :secret_access_key => secret_access_key,
-
-        :logger            => $aws_log,
-        :log_level         => :debug,
-        :log_formatter     => AWS::Core::LogFormatter.new(AWS::Core::LogFormatter.default.pattern.chomp),
+      ManageIQ::Providers::Amazon::CloudManager.raw_connect_v2(
```

Comment:
Yeah, don't do this. The options are the same, but it's the wrong abstraction... they're two quite unrelated things that both happen to talk to AWS. We'll be able to revisit this split while we're gemifying the providers, and can decide where this lives. Right now, a dependency from an authenticator on a provider class would be.. inconvenient.